Mathlib/Data/Quot.lean

```diff
@@ -299,15 +299,19 @@ theorem Quot.eq {α : Type*} {r : α → α → Prop} {x y : α} :
 theorem Quotient.eq [r : Setoid α] {x y : α} : Quotient.mk r x = ⟦y⟧ ↔ x ≈ y :=
   ⟨Quotient.exact, Quotient.sound⟩
 
-theorem forall_quotient_iff {α : Type*} [r : Setoid α] {p : Quotient r → Prop} :
-    (∀ a : Quotient r, p a) ↔ ∀ a : α, p ⟦a⟧ :=
-  ⟨fun h _ ↦ h _, fun h a ↦ a.induction_on h⟩
-#align forall_quotient_iff forall_quotient_iff
+lemma Quotient.forall {α : Sort*} {s : Setoid α} {p : Quotient s → Prop} :
+    (∀ a, p a) ↔ ∀ a : α, p ⟦a⟧ :=
```

Comment:
I think we should have a version for `Quotient.mk s a` too.

Reply:
Isn't `⟦a⟧` just the notation for `Quotient.mk _ a`?

Reply:
No, I think it's notation for `Quotient.mk' a`. Confusingly the meanings of `mk` and `mk'` swapped in Lean4

Reply:
https://github.com/leanprover-community/mathlib4/blob/master/Mathlib/Data/Quot.lean#L223-L224